optimum/onnxruntime/__init__.py

```diff
@@ -57,6 +59,14 @@
     ],
 }
 
+try:
+    if not is_diffusers_available():
+        raise OptionalDependencyNotAvailable()
+except OptionalDependencyNotAvailable:
+    pass
+else:
+    _import_structure["modeling_diffusion"] = ["ORTStableDiffusionPipeline"]
```

Comment:
would
```suggestion
try:
    if not is_diffusers_available():
        raise OptionalDependencyNotAvailable()
    _import_structure["modeling_diffusion"] = ["ORTStableDiffusionPipeline"]
except OptionalDependencyNotAvailable:
    pass
```

be different?